frontend_tests/node_tests/composebox_typeahead.js

```diff
@@ -187,6 +188,30 @@ global.user_groups.add(backend);
     expected_value = '{ :octopus: ';
     assert.equal(actual_value, expected_value);
 
+    // popular emoji
+    fake_this.completing = 'emoji';
```

Comment:
Would it make more sense to test this like done here: https://github.com/zulip/zulip/blob/5ccd3aa71b684c55ad4be58d8ea307fa0bb3cbdd/frontend_tests/node_tests/composebox_typeahead.js#L590 ?

Reply:
If we're using [options.sorter()](https://github.com/zulip/zulip/blob/5ccd3aa71b684c55ad4be58d8ea307fa0bb3cbdd/frontend_tests/node_tests/composebox_typeahead.js#L590) to test for [popular emoji](https://github.com/utkarsh23/zulip/blob/e8442eabd437b3ff6d6714cb26142acabd3127a8/frontend_tests/node_tests/composebox_typeahead.js#L191) alone, shouldn't we use [options.sorter()](https://github.com/zulip/zulip/blob/5ccd3aa71b684c55ad4be58d8ea307fa0bb3cbdd/frontend_tests/node_tests/composebox_typeahead.js#L590) for others also like [emoji](https://github.com/utkarsh23/zulip/blob/e8442eabd437b3ff6d6714cb26142acabd3127a8/frontend_tests/node_tests/composebox_typeahead.js#L167) etc? I can work on this, but wouldn't it be better if I create a new PR for this? What do you think?

Reply:
@utkarsh23 Oh sorry I missed your comment here! I think you are misunderstanding what we are testing there. There we are actually testing the text that will be inserted if the typeahead is selected for the current input while in our case we want to test the ordering of the items in typeahead for the current input. Does it make sense now?